MKSOutputDevice.py

```diff
@@ -510,38 +530,7 @@ def isBusy_error_message(self):
     def is_contains_chinese(self, strs):
         return False
 
-    def closeMDialog(self):
-        if self._mdialog:
-            self._mdialog.close()
-
-    def renameupload(self, filename):
-        if self._mfilename and ".g" in self._mfilename.text().lower():
-            filename = filename[:filename.
-                                rfind("/")] + "/" + self._mfilename.text()
-            if self._mfilename.text() in self.sdFiles:
-                self.show_exists_dialog(filename,
-                                        lambda: self.uploadfunc(filename))
-                return
-            if len(filename[filename.rfind("/") + 1:]) >= 30:
-                self.show_to_long_dialog(filename,
-                                         lambda: self.renameupload(filename))
-                return
-            if self.is_contains_chinese(filename[filename.rfind("/") + 1:]):
-                self.show_contains_chinese_dialog(
-                    filename, lambda: self.renameupload(filename))
-                return
-            if self.isBusy():
-                self.isBusy_error_message()
-                return
-            self._mdialog.close()
-            if self._progress_message:
```

Comment:
when you refactored the code this line was removed. I'd add more logging for new code, and move back it the right place `_progress_message` part of code if it's needed.